rust/operator-binary/src/crd/druidconnection.rs

```diff
@@ -0,0 +1,139 @@
+use serde::{Deserialize, Serialize};
+use snafu::Snafu;
+use stackable_operator::{
+    k8s_openapi::{apimachinery::pkg::apis::meta::v1::Time, chrono::Utc},
+    kube::{CustomResource, ResourceExt},
+    schemars::{self, JsonSchema},
+};
+use stackable_versioned::versioned;
+
+#[derive(Snafu, Debug)]
+#[allow(clippy::enum_variant_names)]
+pub enum Error {
+    #[snafu(display("{druid_connection} is missing a namespace, this should not happen!"))]
+    NoNamespace { druid_connection: String },
+}
+type Result<T, E = Error> = std::result::Result<T, E>;
+
+#[versioned(version(name = "v1alpha1"))]
+pub mod versioned {
+    /// The DruidConnection resource can be used to automatically deploy a Druid datasource in Superset.
+    /// Learn more about it in the [Superset operator usage guide](DOCS_BASE_URL_PLACEHOLDER/superset/usage-guide/connecting-druid).
+    #[derive(Clone, CustomResource, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+    #[versioned(k8s(
+        group = "superset.stackable.tech",
+        plural = "druidconnections",
+        status = "DruidConnectionStatus",
+        namespaced,
```

Comment:
note: This works, but if we already version the status, we should explicitly use a version here.

```suggestion
        plural = "druidconnections",
        status = "v1alpha1::DruidConnectionStatus",
        namespaced,
```

Reply:
https://github.com/stackabletech/superset-operator/pull/597/commits/6281965f50b919b4329fc65abc5457ea1838bc7b